hieradata/node/ts-csc-generic-01.cp.lsst.org.yaml

```diff
@@ -30,3 +30,15 @@ nfs::client_mounts:
     share: "lsstdata"
     server: "nfs1.cp.lsst.org"
     atboot: true
+  /mnt/dmcs/data:
```

Comment:
I presume that changing this would require changing the mounts inside containers.  This PR is more or less puppetifying what already exists rather than trying to rationalize.